_data/translation-management-systems.yaml

```diff
@@ -0,0 +1,669 @@
+---
+#- id: [Lowercase, words separated by hyphen]
```

Comment:
This is good.

We should probably use https://json-schema.org/.

Then it can be automatically enforced.

It works fine with YAML, not just JSON.